### DISCUSSION TITLE
Fling gesture on web always finalizes with success false. Works fine with legacy implementation.

### DISCUSSION THREAD

**Discussion by landabaso** [software-mansion/react-native-gesture-handler#2726] [General]
Fling gesture on web always finalizes with `success = false`. Same code/gesture works fine in iOS & Android.

I'm running expo with "react-native-gesture-handler": "~2.12.0".

The weird thing is it finalizes correctly (`success === true`) when using `enableLegacyWebImplementation()`.

Is Fling supported on web? Thanks!

**Comment by m-bert** [comment]
Hi! Which version of gesture handler are you using? Fling is supported on web, but we are still trying to figure out how to make it work correctly. 

To activate, `Fling` has to meet minimum threshold. Until `2.14.1` it had to travel `160px` (which is basically impossible). Now the minimum distance is set to `32px` ([PR](https://github.com/software-mansion/react-native-gesture-handler/pull/2709)). Even though it is lower, we still aren't happy with how it works and we do have plans to change that. Our plan is to change `Fling` so that it will use velocity to determine whether it should activate or not.

**Comment by m-bert** [comment]
Hi! A while ago we've changed `Fling` implementation, so that now it uses `velocity` instead of `distance` as an activation criterion.

I've just checked that the problem you've described no longer occurs. Therefore, I'm closing this discussion. Feel free to re-open it if it was not solved.
